sos/report/plugins/microovn.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2018 Mark Michelson <mmichels@redhat.com>
```

Comment:
You probably don't need to add this copyright line. It's coming from the other ovn plugins, right?

Reply:
I just wanted to honor the original author's code since this is based off ovn_central plugin.